spec/support/service_spec_helper.rb

```diff
@@ -4,6 +4,8 @@ def with_modified_env(options, &block)
   end
 
   def default_headers
-    { 'x-rh-identity' => encoded_user_hash }
+    { 'x-rh-identity'            => encoded_user_hash,
+      'x-rh-insights-request-id' => 'gobbledygook',
+      'original_url'             => 'some_url' }
```

Comment:
Can you give an example of what value is normal here?  Is it a fully qualified url or just a partial?

Is this value ever parse, should we put something closer to a real expected text here?

Reply:
Thanks @lindgrenj6.

@mkanoor I am fine with the value if it is really just a place holder, just wanted to check.

Reply:
@gmcculloug these are just place holders since it represents the headers that we forward between services.